lib/amazon/bucket.rb

```diff
@@ -14,6 +14,15 @@ def initialize(bucket_name, access_key_id, access_secret_key, region)
 
     attr_reader :region
 
+    def put_object_acl(bucket_name, object_key, acl)
```

Comment:
It looks like this will work on existing objects, do we need to be able to upload an ACL with an object on eg `upload_file` or `upload_data`?

Reply:
When I looked at the SDK docs, it didn't look to me like there were any options on `upload_file` to add an ACL. This was all I could find.